### PR TITLE
Updated the meshkit error in mesh_ops_handler

### PR DESCRIPTION
**Pull request by sayantan1413**
Signed-off-by: sayantan1413 <sayantanbose2001@gmail.com>

**Description**

Updated the meshkit error in the mesh_ops_handler

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: handlers/mesh_ops_handlers.go [hunk 1]
```diff
@@ -108,50 +106,46 @@ func (h *Handler) addAdapter(ctx context.Context, meshAdapters []*models.Adapter
...
-		err = errors.Wrapf(err, "Error creating a mesh client.")
-		logrus.Error(err)
-		// http.Error(w, "Unable to connect to the Mesh adapter using the given config, please try again", http.StatusInternalServerError)
-		return meshAdapters, err
+		h.log.Error(ErrMeshClient)
+		// http.Error(w, ErrMeshClient.Error(), http.StatusInternalServerError)
+		return meshAdapters, ErrMeshClient
 	}
-	logrus.Debugf("created client for adapter: %s", meshLocationURL)
+	h.log.Debug("created client for adapter: %s", meshLocationURL)
```
Comment (dhruv0000) [inline handlers/mesh_ops_handlers.go:131]: `h.log.Debug` uses `logrus.Log` internally, and not `logrus.Logf`.

you might want to rewrite it as
```golang
h.log.Debug("created client for adapter: ", meshLocationURL)
```

File: handlers/mesh_ops_handlers.go [hunk 2]
```diff
@@ -309,22 +299,20 @@ func (h *Handler) AdapterPingHandler(w http.ResponseWriter, req *http.Request, p
...
+		h.log.Error(ErrInvalidK8SConfig)
+		http.Error(w, ErrInvalidK8SConfig.Error(), http.StatusBadRequest)
 		return
 	}
 
 	mClient, err := meshes.CreateClient(req.Context(), prefObj.K8SConfig.Config, prefObj.K8SConfig.ContextName, meshAdapters[aID].Location)
 	if err != nil {
-		logrus.Errorf("Error creating a mesh client: %v.", err)
-		http.Error(w, "Adapter could not be pinged.", http.StatusBadRequest)
+		http.Error(w, ErrMeshClient.Error(), http.StatusBadRequest)
```
Comment (dhruv0000) [inline handlers/mesh_ops_handlers.go:315]: Log the Error?

**Review by dhruv0000 — CHANGES_REQUESTED** [review]
Great work @sayantan1413 :smile: .

A few suggestions.

File: handlers/mesh_ops_handlers.go (hunk 2 above)
Comment (sayantan1413) [reply handlers/mesh_ops_handlers.go:315]: Ah sorry I forgot to do that I am doing it .

File: handlers/mesh_ops_handlers.go (hunk 1 above)
Comment (sayantan1413) [reply handlers/mesh_ops_handlers.go:131]: okay so all the h.log.Debug should be changed right?

**Pull request closed without merging**